joule-labs/joule-extension
# Loop Swap

Closes #246 

### Loop Swap Status

Added v1/loop/swap to Loop Http Library. Added Swap History to Home Page tab panes.

### Steps to Test

1. Install LND 0.10.0-beta.rc2
2. Intall LOOP 0.5.1-beta 
3. Set up loop on Loop Out/In settings
4. Initiate Loop / Out In
5. View status on "Swaps" tab
6. Click on Swap Row to see Swap Details

![image](https://user-images.githubusercontent.com/13033037/79599694-f704a700-80b3-11ea-8ab8-04a87bae7ae8.png)

![image](https://user-images.githubusercontent.com/13033037/79599755-0e439480-80b4-11ea-84b9-12a4be832077.png)
